rotkehlchen/chain/ethereum/modules/uniswap/v3/decoder.py

```diff
@@ -434,16 +441,17 @@ def _maybe_decode_v3_deposit_or_withdrawal(
                 event.event_type = to_event_type[0]
                 event.event_subtype = to_event_type[1]
                 event.counterparty = CPT_UNISWAP_V3
+                event_asset = event.asset.resolve_to_crypto_asset()
                 event.notes = notes.format(
                     amount=event.balance.amount,
-                    asset=resolved_assets_and_amounts[0].asset.symbol,
+                    asset=event_asset.symbol,
```

Comment:
Why this change? Isn't `event_asset` and `resolved_assets_and_amounts[0].asset` the same thanks to the if check? This way you don't have to do the resolution you added up there.

Reply:
No because the asset could be ETH and then it would differ from the WETH that is resolved before. In `resolved_assets_and_amounts` we only have EVm tokens and in the events we can have evm tokens or ETH  I can add a if check and do the resolution only if it is eth the asset

Reply:
Since this is already seen in the utility function you made can you modify it to return this information too? We know ETH's symbol so no need to resolve in that case.